tlslite/tlsrecordlayer.py

```diff
@@ -833,6 +846,9 @@ def _getNextRecord(self):
             # application data isn't made out of messages, pass it through
             if header.type == ContentType.application_data:
                 yield (header, parser)
+            # heartbeat message isn't made out of messages, too
+            elif header.type == ContentType.heartbeat:
+                yield (header, parser)
```

Comment:
d'oh, I was thinking this is in `_getMsg`, yes, for `_getNextRecord` it's exactly what we need